packages/button-theme/stories/Fieldset.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react/types-6-0';
+import Fieldset from '../src/Fieldset';
+import Input from '../src/Input';
+import { HtmlOnlyWrapper } from '../../../stories/helpers';
+
+export default {
+  title: 'Fieldset',
+  component: Fieldset,
+  argTypes: {
+    size: {
+      control: {
+        type: 'select',
+        options: ['small', 'medium', 'large'],
+      },
+      description: 'The size of the checkbox',
+    },
+    onClick: { action: 'clicked' },
+    title: {
+      description: "The title to display in the fieldset's legend.",
+    },
+    disabled: {
+      description: 'Indicates whether the field is disabled.',
+    },
+    required: {
+      description: 'Indicates whether the field is required.',
+    },
```

Comment:
Should we add the fullWidth prop here?

Reply:
I'm going to make a [tech debt card](https://github.com/button-inc/service-development-toolkit/issues/527) to look at this in other components--I suspect there are other stories that are missing props too